public/styles.css

```diff
@@ -2,6 +2,49 @@ body {
   background-color: rgb(201, 198, 198);
 }
 
+:root {
+  --star-size: 15px;
+  --star-color: gray;
+  --star-background: orange;
+  --bar-size: 15px;
+  --bar-color: gray;
+  --bar-background: orange;
+}
+
+/* STARS SECTION - to use, simply add the stars class and pass in a number into style={{ '--rating': NUM }} */
+
+.stars:before {
+  content: '★★★★★';
+  letter-spacing: 0;
+  background: linear-gradient(90deg, var(--star-background) var(--percent), var(--star-color) var(--percent));
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.stars {
+  --percent: calc(var(--rating) / 5 * 100%);
+  display: inline-block;
+  font-size: var(--star-size);
+  font-family: Times; /* make sure ★ appears correctly*/
+  line-height: 1;
+}
+
+/* BAR SECTION - to use, simply add the bar class and pass in a number into style={{ '--rating': NUM }} */
```

Comment:
Wow this stuff is REALLY cool and looks fairly simple to implement, I'm excited to get this added to our modules!